lucene/backward-codecs/src/java/org/apache/lucene/backward_codecs/lucene80/Lucene80DocValuesFormat.java

```diff
@@ -163,6 +163,11 @@ public Lucene80DocValuesFormat(Mode mode) {
     this.mode = Objects.requireNonNull(mode);
   }
 
+  /**
+   * Note: although this format is only used on older versions, we need to keep the write logic in
+   * addition to the read logic. It's possible for doc values on older segments to be written to
+   * through in-place doc values updates.
```

Comment:
`in-place` might be a bit misleading since doc-value updates are not in-place but create a new generation of a segment?
```suggestion
   * through doc values updates.
```

Reply:
Agreed, in-place can mean too many things...